Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,138 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import re
+from DIRAC import S_OK, DError
+import errno
+from DIRAC.Core.Utilities import Subprocess
+
+def executeRabbitmqctl(arg, *argv):
+  """Executes RabbitMQ administration command.
+    It uses rabbitmqctl command line interface.
+    For every command the -q argument ("quit mode")
+    is used, since in some cases the output must be processed,
+    so we don't want any additional informations printed.
+  Args:
+    arg(str): command recognized by the rabbitmqctl.
+    argv(list): optional list of string parameters.
+  Returns:
+    S_OK:
+    Derror:
+
+  """
+  command =['sudo','/usr/sbin/rabbitmqctl','-q', arg] + list(argv)
```

Comment:
I have no idea. Why calling sudo on script is better?

Btw his module  should be in the second PR so I will move it there.